.jenkins/pytorch/win-test.sh

```diff
@@ -12,8 +12,13 @@ if [[ ${JOB_NAME} == *"develop"* ]]; then
   export IMAGE_COMMIT_TAG=develop-${IMAGE_COMMIT_TAG}
 fi
 
+export PYTORCH_COLLECT_COVERAGE=1
```

Comment:
Could this be enabled only for `pytorch_windows_vs2019_py36_cuda10.1_test1` config?

Reply:
It depends on the answer to the previous question. If the answer is yes, then we should do it.